cirq-google/cirq_google/devices/grid_device.py

```
@@ -50,32 +50,16 @@ def _validate_device_specification(proto: v2.device_pb2.DeviceSpecification) ->
                         " which is not in valid_qubits."
                     )
 
-        # Symmetric and asymmetric targets should not have repeated qubits.
-        if (
-            target_set.target_ordering == v2.device_pb2.TargetSet.SYMMETRIC
-            or target_set.target_ordering == v2.device_pb2.TargetSet.ASYMMETRIC
-        ):
+        # Symmetric targets should not have repeated qubits.
+        if target_set.target_ordering == v2.device_pb2.TargetSet.SYMMETRIC:
```

Should we raise an error if target set is not symmetric?